modules/expandable-panel/ExpandablePanel.story.mdx

```diff
@@ -0,0 +1,308 @@
+      <td>
+        This style affects the header's icon (down-arrow) if no custom header
+        renderer is provided
+      </td>
+    </tr>
+    <tr>
+      <td></td>
+      <td>.disabled</td>
+      <td>Style to be applied if panel is disabled</td>
+    </tr>
+    <tr>
+      <td></td>
+      <td>.expanded</td>
+      <td>Style to be applied if panel is expanded</td>
+    </tr>
+    <tr>
+      <td>headerTitle</td>
+      <td></td>
+      <td>
+        This style affects the header's title if no custom header renderer is
+        provided
+      </td>
+    </tr>
+    <tr>
+      <td></td>
+      <td>.disabled</td>
+      <td>Style to be applied if panel is disabled</td>
+    </tr>
+    <tr>
+      <td></td>
+      <td>.expanded</td>
+      <td>Style to be applied if panel is expanded</td>
+    </tr>
+    <tr>
+      <td>panel</td>
+      <td></td>
+      <td>This style affects the ExpandablePanel's root element</td>
+    </tr>
+    <tr>
+      <td></td>
+      <td>.disabled</td>
+      <td>Style to be applied if panel is disabled</td>
+    </tr>
+    <tr>
+      <td></td>
+      <td>.expanded</td>
+      <td>Style to be applied if panel is expanded</td>
+    </tr>
+  </tbody>
+</table>
+
+# ExpandablePanels
+
+`ExpandablePanel`s can be grouped in an `ExpandablePanels` component. Multiple `ExpandablePanel`s can be opened simultaneously by default.
+However, if the `allowMultipleExpandedPanels` prop is passed as `false`, `ExpandablePanels` can mimic the behaviour of an Accordion, where expanding
+one panel will collapse the other.
+This component also passes other props to its children, e.g. `onClick` or `disabled`.
+
+<Preview>
+  <Story name="ExpandablePanels">
+    <ExpandablePanelsStory />
+  </Story>
+</Preview>
+
+### Disabled
+
+<Preview>
+  <Story name="disabled ExpandablePanels">
+    <DisabledExpandablePanelsStory />
+  </Story>
+</Preview>
+
+### Initially expanded
+
+<Preview>
+  <Story name="expanded ExpandablePanels">
+    <ExpandedPanelsStory />
+  </Story>
+</Preview>
+
+### Accordion behaviour (`allowMultipleExpandedPanels === false`)
+
+<Preview>
+  <Story name="accordion">
+    <AccordionStory />
+  </Story>
+</Preview>
+
+### Available props
+
+<table width="100%">
+  <thead>
+    <tr>
+      <th>Prop Name</th>
+      <th>Type</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>allowMultipleExpandedPanels (optional)</td>
+      <td>boolean</td>
+      <td>Flag indicating whether there is an error present in radio button</td>
+    </tr>
+    <tr>
+      <td>children (optional)</td>
+      <td>JSX.Element[]</td>
+      <td>A React element to provide a custom input for styling purposes</td>
+    </tr>
+    <tr>
+      <td>disabled (optional)</td>
+      <td>boolean</td>
+      <td>
+        Flag indicating whether user can interact with any of the
+        ExpandablePanel components
+      </td>
+    </tr>
+    <tr>
+      <td>initialExpandedPanelIndex (optional)</td>
+      <td>number</td>
```

Comment:
We could change this to be `number | number[]` to allow setting multiple initial expanded panels, this way we don't have a situation where this prop is "invalid". What do you think?

Reply:
I also considered that.
My reasoning was that this prop would be used if the `allowMultipleExpandedPanels` prop would be used as well. So if ÈxpandablePanels` were to act as an accordion, `initialExpandedPanelIndex ` is passed. Otherwise, the `initialExpanded` prop can be passed to each of the child components.

```
initialExpanded: allowMultipleExpandedPanels
  ? initialExpandedPanelIndex === index || child.props.initialExpanded
  : initialExpandedPanelIndex === index
```

That being said, do you still think your suggestion should still be implemented? If so, we would also have to check if it is an accordion and only expand the first index of the array?

Reply:
I'd say that if we implement this then the user is responsible for correctly using the array prop

Reply:
For simplicity sake I suggest leaving it as is. If you feel strongly that we should add this change, I can add it.